src/Symfony/Installer/SelfUpdateCommand.php

```diff
@@ -98,16 +97,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 ->cleanUp()
             ;
         } catch (IOException $e) {
+            if ($output->isVeryVerbose()) {
+                echo $e->getMessage();
```

Comment:
You could just add the `$e` var as third argument of the `RuntimeException` thrown right after it